resources/views/navigation.blade.php

```diff
@@ -0,0 +1,22 @@
+    <head>
+        <link href="{{ URL::asset('css/navigation.css') }}" rel="stylesheet"/>
+    </head>
+    <div class="header">
```

Comment:
With .header being removed for being empty, this div should not have it as a class anymore

Reply:
removed